cookies-modal.html

```diff
@@ -1,10 +1,17 @@
-<div id="cookies-modal-container">
-  <div id="cookies-modal">
-    <div id="cookies-title">
-      🍪 Improve your experience by accepting our <a href="#YOUR-PRIVACY-POLICY-PAGE" target="_blank">privacy policy</a>.
-    </div>
-    <div id="cookies-modal-buttons">
-      <button id="accept-cookies-button" onclick="document.getElementById('cookies-modal').className='cookies-d-none';">Accept</button> <button onclick="document.getElementById('cookies-modal').className='cookies-d-none';">Reject</button>
-    </div>
-  </div>
-</div>
\ No newline at end of file
+<dialog id="cookies-modal">
+	<header>
+		🍪 Improve your experience by accepting our
+		<a href="#YOUR-PRIVACY-POLICY-PAGE" target="_blank">privacy policy</a>.
+	</header>
+	<footer>
+		<button
+			id="accept-cookies-button"
+			onclick="this.closest('dialog').close()"
```

Comment:
Instead of looking in the entire document for the id, we just look for the parent `dialog` and close it.